sbtc-cli/src/commands/deposit.rs

```diff
@@ -68,13 +68,12 @@ pub fn build_deposit_tx(deposit: &DepositArgs) -> anyhow::Result<()> {
 
 	wallet.sync(&blockchain, SyncOptions::default())?;
 
-	let recipient_address =
-		StacksAddress::try_from(deposit.recipient.as_str())?;
+	let stx_recipient = PrincipalData::try_from(deposit.recipient.to_string())?;
```

Comment:
How are you using `try_from()` here if you've implemented `From<String>` and not `TryFrom<String>`?

Reply:
Hm, I thought rust would make it just to work. :-)